fachhochschule-vorarlberg.csl

```diff
@@ -1,33 +1,75 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
   <info>
-    <title>Fachhochschule Vorarlberg (German)</title>
-    <title-short>FHV</title-short>
+    <title>Fachhochschule Vorarlberg Autor-Jahr</title>
```

Comment:
The (German) still needs to be there, please. And is there a note style, or why do you need the "Autor-Jahr"?